projects/vdk-plugins/vdk-impala/src/vdk/plugin/impala/templates/load/fact/insert/README.md

```diff
@@ -12,6 +12,8 @@ This template can be used to load raw data from Data Lake to target Table in Dat
 - target_table    - Data Warehouse table of DW type table, where target data is loaded
 - source_schema   - Data Lake schema, where source raw data is loaded from
 - source_view     - Data Lake view, where source raw data is loaded from
+- check           - (Optional) Callback function responsible for checking the quality of the data
```

Comment:
I think that this should say the callable takes in a parameter which is the table name which contains the data to validate .

Reply:
Yes, makes sense. Improved the check description in scd1, snapshot and insert templates.